tests/unittests/object_mother.py

```diff
@@ -22,8 +22,7 @@ def get_event():
                      location_name='India',
                      topic='Travel & Outdoor',
                      sub_topic='Climbing',
-                     is_sessions_speakers_enabled=True,
-                     type='Camp, Trip, or Retreat')
```

Comment:
@niranjan94  I removed this temporarily for travis to pass. I will make a proper test with event_type because this `type` is being used in most of the files, so either I will make an issue to fix the test or solve it here #3695